combination_test/test_assert.sh

```
@@ -32,3 +32,35 @@ assert() {
 	echo
 }
 
+assert_error_handle() {
+	EXPECTED_ERRORS_FILE="./combination_test/resources/expected_syntax_error_msg.txt"
+	printf '%-30s:' "\"$2\""
+	# exit status
+	echo -n -e "$2" | bash --norc --noprofile 2>cmp 1>&-
```

Shellcheck (0.10.0)

[warning] Redirecting to/from command name instead of file. Did you want pipes/xargs (or quote to ignore)?

(SC2238)